contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -70,6 +72,9 @@ const (
 	defaultReconcileInterval = 300    // 5m default task reconciliation interval
 	defaultReconcileCooldown = 15 * time.Second
 	defaultFrameworkName     = "Kubernetes"
+
+	executorCPUs = podtask.CPUShares(0.25) // initial CPU allocated for executor
```

Comment:
wondering if these values should be "finely tuneable" via the scheduler config file

thoughts?